src/command.c

```diff
@@ -309,10 +309,10 @@ static int dv_convert_conv_v0(struct device *dev, struct dmp_cmb *cmb,
 
 	// there should be at least one run
 	if (size < sizeof(struct dmp_dv_kcmdraw_conv_v0) - 31 *
-	    sizeof(struct dmp_dv_kcmdraw_conv_v0_run)) {
+			sizeof(struct dmp_dv_kcmdraw_conv_v0_run)) {
 		pr_warn(DRM_DEV_NAME ": Command size is too small: %zu < %zu\n",
-			size, sizeof(struct dmp_dv_kcmdraw_conv_v0) - 31 *
-			sizeof(struct dmp_dv_kcmdraw_conv_v0_run));
+				size, sizeof(struct dmp_dv_kcmdraw_conv_v0) - 31 *
+				sizeof(struct dmp_dv_kcmdraw_conv_v0_run));
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket